optuna/pruners/_wilcoxon.py

```diff
@@ -153,6 +153,14 @@ def prune(self, study: "optuna.study.Study", trial: FrozenTrial) -> bool:
         except ValueError:
             return False
 
+        if len(best_trial.intermediate_values) == 0:
+            warnings.warn(
+                "The best trial has no intermediate values so WilcoxonPruner cannot prune trials."
+                "If you have added the best trial with Study.add_trial, please consider setting "
+                "intermediate_values argument."
```

Comment:
I suppose users mainly call `trial.report` to do so rather than directly set `intermediate_values`. So it would be more user-friendly to mention `trial.report` here. What do you think?

Reply:
I thought most users would call `create_trial` to create trials, but is it intended to call `study.ask()` and `study.tell()`?

Reply:
Ah, sorry. I misunderstood the warning message. Thank you for your comment!